package.json

```diff
@@ -18,6 +18,14 @@
     "test": "jest --passWithNoTests",
     "prepublishOnly": "tsc --declaration"
   },
+  "dependencies": {
+    "@types/lodash": "^4.14.149",
+    "eslint-config-prettier": "^6.10.1",
+    "eslint-plugin-prettier": "^3.1.2",
```

Comment:
Are these meant to be `devDependencies`?

Reply:
oof yes, fixing now